Makefile

```diff
@@ -113,5 +113,5 @@ deploy: examples/admission-cert.pem
 check-dependencies:
 	# We need mercurial for bitbucket.org/ww/goautoneg, otherwise dep hangs forever
 	which hg >/dev/null 2>&1 || apt update && apt install -y mercurial
-	$$GOPATH/bin/dep version || (export DEP_RELEASE_TAG=v0.5.0; curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh)
```

Comment:
To make this less annoying, I'd just add a line `export PATH=$(PATH):$(GOPATH)/bin` at the top of the `makefile`